.github/workflows/main.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       if: matrix.build != 'mingw'
     - run: cargo test --features zlib-ng --no-default-features
       if: matrix.build != 'mingw'
+    - run: cargo test --features zlib-rs --no-default-features
+      if: matrix.build != 'mingw'
```

Comment:
we should eventually be able to compile on mingw, but the current release doesn't, and we don't test for it, so disabling for now.